keyboards/zinc/keymaps/toshi0383/keymap.c

```diff
@@ -0,0 +1,163 @@
+   * |      |   :  |   ;  |  "   |  '   |  \   |             | Left | Down | Up   |Right |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |ADJUST|      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      | GUI  |Shift |             |Shift |------|      |      |      |      |
+   * `-----------------------------------------'             `-----------------------------------------'
+   */
+  [_RAISE] = LAYOUT_ortho_4x12( \
+  _______, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,           KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  _______, \
+  _______, KC_COLN, KC_SCLN, KC_DQT, KC_QUOT, KC_BSLS,         KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, _______, _______, \
+  _______, _______, _______, _______, _______, _______,        ADJUST, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, KC_LGUI, KC_LSFT,        KC_LSFT, _______, _______, _______, _______, _______ \
+  ),
+
+  /* Adjust (Lower + Raise)
+   * ,-----------------------------------------.             ,-----------------------------------------.
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |      |             |      |      |      |      |      |      |
+   * |------+------+------+------+------+------|             |------+------+------+------+------+------|
+   * |      |      |      |      |      |RESET |             |------|      |      |      |      |      |
+   * `-----------------------------------------'             `-----------------------------------------'
+   */
+    [_ADJUST] =  LAYOUT_ortho_4x12( \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______, _______, _______, \
+      _______, _______, _______, _______, _______, RESET,  _______, _______, _______, _______, _______, _______ \
+      ),
+
+};
+
+// define variables for reactive RGB
+bool TOG_STATUS = false;
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_QWERTY);
+      }
+      return false;
+      break;
+
```

Comment:
You don't need Lines 113-119. They should be deleted.

Reply:
0299bb0c39851c4b834b4a3ad31fde9cb207bed6